contributing/FORAMTTING-CODE.md

```diff
@@ -0,0 +1,169 @@
+      Cmdlet          ConvertFrom-SecureString                           3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          ConvertTo-SecureString                             3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Get-Acl                                            3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Get-AuthenticodeSignature                          3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Get-CmsMessage                                     3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Get-Credential                                     3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Get-ExecutionPolicy                                3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Get-PfxCertificate                                 3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          New-FileCatalog                                    3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Protect-CmsMessage                                 3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Set-Acl                                            3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Set-AuthenticodeSignature                          3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Set-ExecutionPolicy                                3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Test-FileCatalog                                   3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Unprotect-CmsMessage                               3.0.0.0    Microsoft.PowerShell.Security
+      ```
+
+The **Output** code label is not an official "language" supported by the syntax highlighting system.
+However, this label is useful because OPS adds the "Output" label to the code box on the web page.
+And this "Output" code box has no syntax highlighting.
+
+## Understanding the COPY button
+
+OPS also adds a **Copy** button that copies the contents of the code block to the clipboard.
+This allows you to quickly paste the code into a script for testing the code example.
+The **Copy** button also removes PowerShell prompts from the text before copying it to the clipboard.
+The **Copy** button uses the following javascript code to remove the prompts.
+
+```javascript
+if (language === 'powershell') {
+  text = text.replace(/\bPS [a-z]:\\>\s?/gi, '');
+}
+```
+
+Note that the regex used only matches simple prompts like "`PS C:\>`".
+Prompts that include subdirectories or paths from other PSDrive providers will not be removed.
+See the general guidelines, in the next section, for the appropriate use of prompts in examples.
+
+## General guidelines
+
+- Code fences
+
+  Markdown allows for indentation to signal code block, however, all code blocks should be contained in a code fence.
```

Comment:
"to signal" sounds a little weird, maybe "to signify" or "Markdown supports indentation for rendering code blocks"?

Reply:
fixed